load.js

```diff
@@ -0,0 +1,128 @@
+import * as yaml from "https://cdn.skypack.dev/js-yaml";
+import showdown from "https://cdn.skypack.dev/showdown";
+
+async function loadModelCard(modelPath, title) {
+    const response = await fetch(modelPath);
+    if (!response.ok) {
+        throw new Error('Failed to fetch model card: ' + response.status);
+    }
+    document.getElementById('modelcardurl').value = modelPath
+
+    const modelCardMarkdown = await response.text();
+    document.getElementById('modelcardraw').value = modelCardMarkdown
+
+    // Extract YAML from Markdown preamble
+    const yamlRegex = /^---\n([\s\S]*?)\n---/;
```

Comment:
Should we also allow yaml at the end of the md file? this would resolve @robbertbos readability issue a little bit when we have huge modelcards